tests/network/hotplug.go

```diff
@@ -453,6 +591,24 @@ func waitForSingleHotPlugIfaceOnVMISpec(vmi *v1.VirtualMachineInstance) *v1.Virt
 	return vmi
 }
 
+func waitForUnplugIfaceApplyOnVMISpec(vmi *v1.VirtualMachineInstance, ifaceName string) *v1.VirtualMachineInstance {
```

Comment:
You should express what you actually wait for instead of abstracting it. This way we understand also how you checked it directly from the tests.

In this case, it looks wrong to me, or I just do not understand it.

We expect the network and interface to be in place, only for the interface `State` to be marked `absent`.

In the future we will expect it to be removed by the controller once the unplug is fully successful.